proposals/0016-simd-application-fees.md

```diff
@@ -0,0 +1,438 @@
+---
+simd: '0016'
+title: Application Fees
+authors:
+  - Godmode Galactus (Mango Markets)
+  - Maximilian Schneider (Mango Markets)
+category: Standard
+type: Core, Fees
+status: Draft
+created: 2022-12-23
+feature:
+---
+
+
+## Summary
+
+This SIMD will discuss additional fees called Application Fees. These fees are
+decided and set by the dapp developer to interact with the dapp. Dapp developers
+then can decide to rebate these fees if a user interacts with the dapp as
+intended and disincentivize the users which do not. These fees will be applied
+even if the transaction eventually fails and are collected on the writable
+account. The owner of the account can do lamport transfer to recover these
```

Comment:
"the writable account" - what writable account? This is the first mention of such a thing in this proposal. A clarification in the summary here would help